Client/Catalog/frontend/components/one-page/one-page.component.html

```diff
@@ -31,7 +31,7 @@ <h1>{{ product.name | translate }}</h1>
         </div>
         <apto-slot-outlet *ngIf="(hideOnePage$ | async)" identifier="one-page-designer"></apto-slot-outlet>
         <div class="description">
-            <h2>Produktdetails</h2>
+            <h2>{{ (contentSnippet$ | async | contentSnippet: 'productDetail.title') || 'Produktdetails' }}</h2>
```

Comment:
Default value for content snippets should be stored in Apto/Catalog/Infrastructure/AptoCatalogBundle/Resources/content-snippets/content-snippets.json